tests/dom.js

```diff
@@ -80,7 +80,7 @@ QUnit.test("Helpers - getClassList() and getRelList()", function(assert) {
   assert.ok(getClassList(document.getElementById('baz')).contains('epsilon'));
   assert.ok(getClassList(document.getElementById('baz')).contains('phi'));
 
-  var elem = document.createElement('span');
+  var elem = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
```

Comment:
I suspect you meant to add cases for this, rather than replacing the existing cases.